### ISSUE TITLE
Add card hint window over chat's messages (lobby and game)

### ISSUE THREAD

**Issue by JayDi85** [magefree/mage#4189]
Found interesting source code in `mage/Mage.Client/src/main/java/mage/client/components/ColorPane.java`

There were code to show up hint window over card names in chat windows. But it wasn't  work many years. Maybe it has bugs and was disabled? I modifies it and add test code -- it works fine!

It's can be very useful: can show card info in chat and in game logs.

Chat messages with `[card name]`:

![shot_171121_233212](https://user-images.githubusercontent.com/8344157/33099418-36280e6a-cf2a-11e7-8ab6-528b47631198.png)

Test animation:

![record_171122_021158](https://user-images.githubusercontent.com/8344157/33099517-8aed0e82-cf2a-11e7-82c3-bfa380f92bbf.gif)

**Comment by Zerrisx** [comment]
I think this has been implemented - do you agree?

**Comment by JayDi85** [comment]
Nope, it's not implemented yet. That issue about custom chat messages -- if you enter card name in chat as discord style (`[card name]`) then xmage can replace it with real card hint like game logs.

**Comment by Zerrisx** [comment]
Oooh, I see.  Gotcha.  Yep, not implemented, but could be cool.
